apis/apis.json

```diff
@@ -55,19 +55,36 @@
       "transport": "grpc+rest"
     },
     {
-      "id": "Google.Apps.Script.Type",
-      "version": "2.1.0",
-      "targetFrameworks": "netstandard2.1;net462",
-      "type": "other",
-      "description": "Version-agnostic types for Apps Script APIs.",
-      "tags": [
-        "gsuite"
-      ],
-      "dependencies": {
-        "Google.Api.CommonProtos": "2.10.0"
-      },
-      "generator": "proto",
-      "protoPath": "google/apps/script/type"
+        "id": "Google.Apps.Script.Type",
```

Comment:
Indentation is changed. Please revert it. Only the newly added changes should be shown in diff.